core-library/src/main/java/org/silverpeas/core/security/authorization/ComponentAccessController.java

```diff
@@ -220,6 +226,7 @@ static class DataManager {
 
     DataManager() {
       controller = OrganizationController.get();
+      removedChecker = create().resetWithCacheSizeOf(1);
```

Comment:
Ok here. The checker life-cycle is bound to the life-cycle of the DataManager instance